integrations/visual_studio.rst

```diff
@@ -4,114 +4,79 @@
 |visual_logo| Visual Studio
 ===========================
 
-Conan can be integrated with **Visual Studio** in two different ways:
+Microsoft Visual Studio is one of the most used IDEs to develop C++ applications and Conan has
+many utilities to work with it. This section serves as a reference to briefly describe some
+of them and link to their own sections, but it will also contain documentation related to
+some utilities that are totally dedicated to Visual Studio itself.
 
-- Using the ``cmake`` generator to create a *conanbuildinfo.cmake* file.
-- Using the ``visual_studio`` generator to create a *conanbuildinfo.props* file.
+There are two different ways that can be used to integrate Conan into Visual Studio, always
+requiring the generators provided by Conan:
 
-With CMake
-----------
+.. toctree::
+   :maxdepth: 2
 
-Use the ``cmake`` generator or ``cmake_multi`` if you are using CMake to machine-generate your Visual Studio projects.
+   Using Conan provided CMake generators <visual_studio/cmake_generators>
+   Using Conan provided Visual Studio generators <visual_studio/visual_studio_generators>
 
-Check the :ref:`generators` section to read about the ``cmake`` generator.
-Check the official `CMake docs`_ to find out more about generating Visual Studio projects with CMake.
 
-However, beware of some current CMake limitations, such as not dealing well with find-packages, because CMake doesn't know how to handle finding both debug and release packages.
+Conan Extension for Visual Studio
+---------------------------------
 
-.. note::
+Thanks to the invaluable help of our community we manage to develop and maintain a free extension
```

Comment:
```suggestion
Thanks to the invaluable help of our community we managed to develop and maintain a free extension
```

Reply:
It is a present continuous... we keep counting on the community to continue the development 😉